second-edition/src/ch01-00-introduction.md

```diff
@@ -1,126 +1,127 @@
+algunos de los grupos más importantes.
+
+### Equipos de Desarrolladores
+
+Rust está probando ser una herramienta de producción por colaborar con grandes
+grupos de desarrolladores con variados niveles de conocimiento en programación
+de sistemas. Los códigos de bajo-nivel son propensos a sufrir una variedad de
+errores menores, los cuales en otros lenguajes de programación sólo pueden ser
+descubiertos tras revisiones cuidadosas de códigos y exámenes extensos por
+desarrolladores experimentados. En Rust, el compilador cumple un rol de portero
+al evitar compilar códigos con esta clase de errores--incluyendo errores de
+concurrencia. Trabajando junto al compilador, el equipo puede usar más tiempo
+en la lógica del programa en vez de persiguiendo errores.
+
+Rust también brinda herramientas de desarrollo contemporáneas al mundo de
+programación de sistemas:
+
+* Cargo, el administrador de dependencias y herramienta de contrucción incluída,
+  hace añadir, compilar y manejar dependencias fácil y consistente dentro del
+  ecosistema de Rust.
+* Rustfmt asegura un estilo de código consistente entre desarrolladores.
+* El Servidor de Lenguaje de Rust soporta la integración de IDE para completación
+  de código y mensajes de error entre lineas.
+
+Usando estas y otras herramientas en el ecosistema Rust, los desarrolladores
+pueden ser productivos mientras escriben códigos al nivel de sistemas.
+
+### Estudiantes
+
+Rust es para estudiantes y personas que estén interesadas en aprender conceptos
+sobre sistemas. Muchas personas han aprendido sobre temas como el desarrollo
+de sistemas operativos a través de Rust. La comunidad está feliz de responder
+preguntas de estudiantes. A través de esfuerzos como este libro, los equipos de
+Rust quieren hacer a los conceptos de sistemas más accesibles para más personas,
+especialmente para aquellos comenzando con programación.
+
+### Compañías
+
+Rust es usado en producción por cientas de compañías, grandes y pequeñas, para
+una variedad de tareas, como herramientas de línea de comandos, servicios web,
+herramientas DevOps, dispositivos embebidos, análisis de audio y video, y
+transcodificación, criptomonedas, bioinformática, motores de búsqueda, aplicaciones
+de cosas en internet, aprendizaje de máquina, e incluso partes mayores del buscador
+web Firefox.
+
+### Desarrolladores Open Source
+
+Rust es para las personas que quieren construir el lenguaje de programación Rust,
+la comunidad, las herramientas de desarrollo, y las librerias. Nos encantaría
+que nos ayudaras con el lenguaje Rust.
+
+### Personas Que Valoran La Velocidad y La Estabilidad
+
+Por velocidad, nos referimos a la velocidad de los programas que Rust le permite
+crear y la velocidad con la cual Rust le permite escribirlos. Las comprobaciones
+del compilador de Rust aseguran estabilidad a través de la añadición de
+características y la refactorización, a diferencia del legado de códigos
```

Comment:
'legado de codigos quebrantables', I would suggest to translate by 'código heredado' or some variation, which is a more precise way to define legacy code

Reply:
will do right away!